kolibri/plugins/safe_html5_viewer/assets/src/views/SafeHtml5RendererIndex.vue

```diff
@@ -87,6 +87,11 @@
       const entryHtmlFile = await zipFile.file(this.entry);
       this.html = entryHtmlFile.toString();
       this.loading = false;
+
+      // Wait for DOM update after loading completes
+      await this.$nextTick();
+      this.setupScrollListener();
```

Comment:
I think this would be better done in a mounted hook rather than created, to ensure the DOM exists?